tests/test_h2h_league.py

```diff
@@ -1,8 +1,91 @@
+import aiohttp
+import pytest
+
+from fpl.models.h2h_league import H2HLeague
+from tests.helper import AsyncMock
+
+h2h_league_data = {
+    "new_entries": {"has_next": False, "number": 1, "results": []},
+    "league": {
+        "id": 829116,
+        "leagueban_set": [],
+        "name": "League 829116",
+        "has_started": True,
+        "can_delete": False,
+        "short_name": None,
+        "created": "2018-08-09T18:10:37Z",
+        "closed": True,
+        "forum_disabled": False,
+        "make_code_public": False,
+        "rank": None,
+        "size": None,
+        "league_type": "c",
+        "_scoring": "h",
+        "ko_rounds": 2,
+        "admin_entry": None,
+        "start_event": 1,
+    },
+    "standings": {"has_next": False, "number": 1, "results": []},
+    "matches_next": {},
+    "matches_this": {},
+}
+
+
+@pytest.fixture()
+async def h2h_league():
+    session = aiohttp.ClientSession()
+    yield H2HLeague(h2h_league_data, session)
+    await session.close()
+
+
 class TestH2HLeague(object):
+    async def test_init(self, loop):
+        session = aiohttp.ClientSession()
+        league = H2HLeague(h2h_league_data, session)
+        assert league._session == session
+        for k, v in h2h_league_data.items():
+            assert getattr(league, k) == v
+        await session.close()
+
     def test_h2h_league(self, loop, h2h_league):
-        assert h2h_league.__str__() == "League 760869 - 760869"
+        assert h2h_league.__str__() == "League 829116 - 829116"
+
+    async def test_get_fixtures_with_known_gameweek_unauthorized(
+        self, loop, h2h_league):
```

Comment:
Should be indented for readability, I think.

Reply:
Indented how?